locales/en/plugin__odf-console.json

```
@@ -1128,6 +1128,12 @@
   "Delete objects": "Delete objects",
   "Actions": "Actions",
   "Create folder": "Create folder",
+  "Failed to delete {{ count }} objects from the bucket. View deletion summary for details._one": "Failed to delete {{ count }} objects from the bucket. View deletion summary for details.",
```

For `._one` shouldn't be the translation in singular?
```suggestion
  "Failed to delete {{ count }} object from the bucket. View deletion summary for details._one": "Failed to delete {{ count }} objects from the bucket. View deletion summary for details.",
```